autogen.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+autoheader
+autoconf
```

Comment:
How about `autoreconf -f -i`? Should do the right thing even if we require `aclocal` or `automake` at some point in the future.

Reply:
autoreconf requires aclocal from automake. I do not have automake on my laptop and autoreconf bombs. I see no reason to require automake.

Reply:
I was not aware that it is now possible to have arbitrary files attach to a release on github. This was the feature missing back when we migrated from sourceforge. 

IMHO it does not make sense to have two downloads ready. Let's wait how this will render in the browser.

and btw. I would have never checked in configure : I was testing you :-)

Reply:
Why is it a problem to install? This is default for GNU autotools.

Reply:
Because I am an old man. I learned GNU the hard way, by handcrafting configure.h . Then I accepted autoconf for automatically detect it by a configure.in.  Than I handcrafted configure.h.in

Now you forced me to use autoheader to generate configure.h.in . accepted: It is helpful.

But I see no use in using automake. IMHO this is one of the worst tools from the GNU Toolchain. It created ton's of conflicts way back in my time where I had to support HPUX, AIX, Solaris ... , that I worked around it all the time. No, I will not use it.
The dependencies to special versions of autoheader, libtoolize and so one are so insane. No, please do not ask me to use automake for msktutil.

Reply:
You should move to cmake if you dislike autotools otherwise accept and embrace the toolset.